backend/app/services/group_member_helper.py

```
@@ -292,3 +292,266 @@ def count_group_members_by_role(db: Session, group_name: str, role: str) -> int:
+            ResourceMember.entity_id == str(user_id),
+            ResourceMember.status == MemberStatus.APPROVED.value,
+        )
+        .all()
+    )
+
+    # Collect namespace info
+    namespace_ids = [m.resource_id for m in direct_memberships]
+    namespaces = (
+        (
+            db.query(Namespace)
+            .filter(Namespace.id.in_(namespace_ids), Namespace.is_active == True)
```

Ruff (0.15.13)

[error] Avoid equality comparisons to `True`; use `Namespace.is_active:` for truth checks

Replace with `Namespace.is_active`

(E712)